core/src/main/java/com/datastax/oss/driver/internal/core/metadata/DefaultMetadata.java

```diff
@@ -20,10 +20,10 @@
 import com.datastax.oss.driver.api.core.metadata.Node;
 import com.datastax.oss.driver.api.core.metadata.TokenMap;
 import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
+import com.datastax.oss.driver.api.core.metadata.token.TokenFactory;
```

Comment:
Please don't do changes like this. They will create merge conflicts later and give us absolutely no value.